Composer/packages/adaptive-form/src/components/fields/FieldSets.tsx

```diff
@@ -5,25 +5,22 @@ import React from 'react';
 import { FieldProps } from '@bfc/extension-client';
 
 import { CollapseField } from '../CollapseField';
-import { getFieldSets } from '../../utils';
+import { getFieldsets } from '../../utils';
 
 import { ObjectField } from './ObjectField';
 
-export const FieldSets: React.FC<FieldProps<object>> = (props) => {
+export const Fieldsets: React.FC<FieldProps<object>> = (props) => {
```

Comment:
I'm not sure why this happens, but exporting this way does not set the component's display name (which is helpful when using the react debugger). Instead do this: `export { Fieldsets }`

Reply:
Weird. I'll update it.